e2e-tests/tests/users/03-read-from-primary.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+timeout: 30
+commands:
+  - script: |-
+      set -o errexit
+      set -o xtrace
+
+      source ../../functions
+
+      data=$(run_psql '\c myapp \\\ SELECT * from myApp;' "postgres:$(get_psql_user_pass users-pguser-postgres)@$(get_psql_user_host users-pguser-postgres)")
+
+      kubectl create configmap -n "${NAMESPACE}" 03-read-from-primary --from-literal=data="${data}"
```

Comment:
Can we check if the data is written here instead of writing it to CM and then checking like that? Just check the `data` here and if it is missing do `exit 1`

Creating a CM with this data and then checking that later feels like overkill imho.

Reply:
The reason we check data with CM is that we use kuttl functionality to compare (together with asserts) and we use it in PS operator also.
But also what Natalia is mentioning, we have a separate step and comparisons are always done the same so it's easier to debug.
If we find a better way I propose to change it everywhere.

Reply:
We already do it in some places, the thing I mentioned, like here in PS:
1. https://github.com/percona/percona-server-mysql-operator/blob/91f6b01d90045646873a609b2b86905277d519a9/e2e-tests/tests/auto-config/07-check-auto-tuning.yaml#LL13C7-L13C24
2. https://github.com/percona/percona-server-mysql-operator/blob/91f6b01d90045646873a609b2b86905277d519a9/e2e-tests/tests/config-router/03-check-config.yaml#L16

Reply:
As for me it doesn't look the same)

Reply:
@inelpandzic @tplavcic I suggest to  discuss it a bit later and fix after GA for all operators (if we decide to do it)

Reply:
I like the configmap approach since it shows what's expected and what's the actual. If we don't have a reason other than it being cumbersome, I suggest keeping the configmap approach.